app/services/users/create_mascot_account.rb

```diff
@@ -0,0 +1,29 @@
+module Users
+  class CreateMascotAccount
+    MASCOT_PARAMS = {
+      email: "mascot@forem.com",
+      username: "mascot",
+      name: "Mascot",
+      profile_image: SiteConfig.mascot_image_url,
+      confirmed_at: Time.current,
+      registered_at: Time.current,
+      password: SecureRandom.hex
+    }.freeze
+
+    def self.call
+      new.call
+    end
+
+    def call
+      return if SiteConfig.mascot_user_id
```

Comment:
It might be useful to know if somehow this service was called and the `mascot_user_id` for some strange reason was _already_ set...maybe we could log it with a warning? Right now will short-circuit silently, but I wonder if there could be a scenario where we'd actually want to know that this service was being called again and again and from the `registrations_controller`... 🤔 

What do you think?

Reply:
I updated it to raise an error.